bin/create-or-update-database-roles.sh

```diff
@@ -32,6 +38,7 @@ CLI_RESPONSE=$(aws lambda invoke \
   --no-cli-pager \
   --log-type Tail \
   --output json \
+  $PAYLOAD_OPTION \
```

Comment:
The linter would prefer this to be double quoted, which doesn't work. Alternatively I could do: 
```
if [ "$ENABLE_PGVECTOR_EXTENSION" = "true" ]; then
  CLI_RESPONSE=$(aws lambda invoke \
    --function-name "$DB_ROLE_MANAGER_FUNCTION_NAME" \
    --no-cli-pager \
    --log-type Tail \
    --output json \
    --payload $(echo -n '"enable-pgvector-extension"' | base64) \
    response.json)
else
  CLI_RESPONSE=$(aws lambda invoke \
    --function-name "$DB_ROLE_MANAGER_FUNCTION_NAME" \
    --no-cli-pager \
    --log-type Tail \
    --output json \
    response.json)
fi```

but that seems too repetitive.